app/Resources/translations/messages.de.xliff

```diff
@@ -199,6 +199,18 @@
                 <source>post.no_posts_found</source>
                 <target>Keine Beiträge gefunden.</target>
             </trans-unit>
+            <trans-unit id="post.created_successfully">
+                <source>post.created_successfully</source>
+                <target>Beitrag erstellt!</target>
+            </trans-unit>
+            <trans-unit id="post.updated_successfully">
+                <source>post.updated_successfully</source>
+                <target>Beitrag bearbeitet!</target>
+            </trans-unit>
+            <trans-unit id="post.deleted_successfully">
+                <source>post.deleted_successfully</source>
+                <target>Beitrag gelöscht!</target>
```

Comment:
I don't speak German so I don't pretend to correct you, but if the original English message is `Post deleted successfully!`, shouldn't the translation be something like `Der Beitrag wurde erfolgreich gelöscht.` instead of `Beitrag gelöscht`?